src/plugins/intel_cpu/src/utils/general_utils.h

```diff
@@ -37,24 +37,14 @@ inline T rnd_up(const T a, const U b) {
     return div_up(a, b) * b;
 }
 
-template <typename T, typename P>
-constexpr bool one_of(T val, P item) {
-    return val == item;
+template <typename T, typename... Args>
+constexpr bool one_of(T val, Args... items) {
+    return ((val == items) || ...);
```

Comment:
```suggestion
    return (false || ... || (val == items));
```
Is there test when Args is empty? Should it fail at compile time or return some value?